docs/.vuepress/tools/copy-assets.mjs

```diff
@@ -0,0 +1,17 @@
+import path from 'path';
+import {copyFileSync} from 'fs'; 
+
+const TARGET_PATH = './.vuepress/public/';
+const filesToMove = [
+  ['../dist/handsontable.full.css', 'handsontable.css'],
+  ['../dist/handsontable.full.js', 'handsontable.js'],
+];
+
+filesToMove.forEach((fileSchema) => {
+  const [from, target] = fileSchema;
+
+  copyFileSync(
+    path.resolve(from),
+    path.resolve(`${TARGET_PATH}/${target}`)
+  );
```

Comment:
I wasn't analyzed anytime before. We should create an issue for next sprint for it.

Reply:
I believe that was one of the reasons why we put `docs` in monorepo. If we can't easily work with develop then the requirement is not met

Reply:
Ok, I suppose that may take more than 5 minutes. Please add an issue and add it into the `New Documentation` project - it will be much easier to manage.

Reply:
It doesn't hold back the initial release. Let's do it at your earliest convenience @Siemienik

Reply:
Reported in https://github.com/handsontable/handsontable/issues/8426.